bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/internal/JdbcConfiguration.java

```diff
@@ -85,13 +85,14 @@ public void updateConfig(Map<Object, Object> config) {
         String url = (String) configuration.get("url");
         Properties parsedURL = StringUtilsExt.parseJdbcURL(url);
 
-        if (user == null || StringUtils.isBlank(user)) {
+        if (user.equalsIgnoreCase("false") || StringUtils.isBlank(user)) {
```

Comment:
So `user`, `password` and `url` can have the string value `"false"`?  That doesn't seem correct.  If they are in fact `null`, the `.equalsIgnoreCase` will cause an NPE.